shotover-proxy/src/transforms/internal_debug_transforms.rs

```diff
@@ -1,23 +1,66 @@
 use crate::error::ChainResponse;
-use crate::message::Messages;
-use crate::transforms::{Transform, Wrapper};
-use anyhow::anyhow;
+use crate::message::{Message, MessageDetails, Messages};
+// use crate::transforms::{
```

Comment:
commented out code.

And should the DebugReturner changes go in there own PR?

Reply:
👍 I'll move them to another PR